### ISSUE TITLE
Bootstrap 4 Support

### ISSUE THREAD

**Issue by jeffcj** [wxsms/uiv#157]
This is a great project. Thank you for the time and effort in developing and maintaining it. It's saved me a lot of time!

With the recent official release of Bootstrap 4.0 will this project be updated/forked to support Bootstrap 4 CSS?

**Comment by smares** [comment]
I really hope a Bootstrap 3 fork will remain actively developed because there are developers who still have to support browsers that lack support for flexbox.

**Comment by wxsms** [comment]
Hi there. thank you for using this project! 

Currently there're no plans on Bootstrap 3->4 upgrading, Because there's already one exist: https://github.com/bootstrap-vue/bootstrap-vue. However, if you found some really good reasons that this lib HAVE to upgrade, I will think about it.

Anyway, Bootstrap 3 will always be supported here, with all latest Vue.js upgrades.

**Comment by ggedde** [comment]
I would also like to see a Bootstrap 4 version.
Unfortunately, https://github.com/bootstrap-vue/bootstrap-vue only covers bootstrap and does not account for anything extra.  Like Date Picker, Time Picker, etc.  Where your repo seems to be more thought out for full development purposes.  Like not including every single thing that bootstrap comes with, but just the necessity for an App.  Which I like.

I also like that there is no prefix.

**Comment by jleaders** [comment]
Also it would be nice when upgrading from bootstrap 3 to bootstrap 4 to not have to rewrite as much of the existing app code. I would add a link to this issue in the Readme.md, because longevity of a framework is a major concern when it it selected. If you are suggesting that no bootstrap 4 fork will be made, this framework has an expiration date. 
However, this looks like a nice framework so my vote is that a fork for 4 would be started, so people can start contributing. I'd keep it in this same repo, just make it on a different branch.

**Comment by wxsms** [comment]
@jleaders 

> I would add a link to this issue in the Readme.md

I think it's good to have more discussions on this issue. PR is welcome. If more people wanted the v4 upgrade, I will start working on it.

**Comment by nam-co** [comment]
Thanks for reopening this, hope you'll consider it, great package, specially Typeahead and Notification

good luck

**Comment by jesperhoy** [comment]
What I really like about this project is that it is lean - because you DON'T wrap every single Bootstrap class / component - unlike bootstrap-vue

I don't need "help" writing HTML with Bootstrap classes. I just want the JQuery stuff replaced with Vue.js. What you have here is perfect.

It would be so awesome to have the same for Bootstrap 4.

**Comment by unitedworx** [comment]
I agree! It will be great if uiv releases a fork for bootstrap 4. Yours is a much better aproarch imho compared to bootstrap-vue!

I consider it very bad that other libraries try to replace HTML, I just need the jquery bits replaced with vue components!

**Comment by olemarius** [comment]
I'm planning to start using vue in a fairly big web project where we're currently using bootstrap 3. If we could migrate from non-vue boostrap to using uiv bootstrap 3 and then easily upgrade to a future uiv bootstrap 4 it'd save us a ton of work :-)
